tool/practice.re

```diff
@@ -75,7 +74,7 @@ static PRACTICE_COMPONENT = Component {
     on_level_change: fn(old: int, new: int) {},
     on_reset: fn(old: int, new: int) {
         press_buttons_until(CURRENT_PRACTICE.button);
-        Tas::apply_map(CURRENT_PRACTICE_MAP);
+        Tas::set_all_cluster_speeds(700.);
```

Comment:
That doesn't work when map-editing and practice states are combined. If the map modifies the rising speed of any cluster, this change will reset that change. Instead, a new function needs to be added, that given a map applies only all cluster speeds.